kafka-rest/src/test/java/io/confluent/kafkarest/resources/v3/ProducerMetricsTest.java

```diff
@@ -209,4 +214,64 @@ public void testTenantTag() throws Exception {
     String tenantId = beanNames.stream().iterator().next().getKeyPropertyList().get("tag");
     assertEquals("value", tenantId);
   }
+
+  @Test
+  public void testMultipleSensors() throws Exception {
+
+    ProducerMetrics producerMetrics1 =
+        new ProducerMetrics(config, ImmutableMap.of("tag", "value1"));
+    Map<String, String> tags2 = new HashMap<>();
+    tags2.put("tag", "value2");
+    tags2.put("otherTag", "otherValue2");
+    ProducerMetrics producerMetrics2 = new ProducerMetrics(config, tags2);
+
+    producerMetrics1.recordRequest();
+    producerMetrics2.recordRequest();
+    producerMetrics2.recordRequest();
+
+    MBeanServer mBeanServer = ManagementFactory.getPlatformMBeanServer();
+    Set<ObjectName> beanNames = mBeanServer.queryNames(new ObjectName(METRICS_SEARCH_STRING), null);
+    assertEquals(3, beanNames.size()); // don't forget the one made in setup
+    Iterator<ObjectName> i = beanNames.stream().iterator();
+    String tenantId = i.next().getKeyPropertyList().get("tag");
```

Comment:
nit: Shouldn't this value also be part of an assertion? It should have the value `value`, to confirm it's the one made during setup, right?

Reply:
I tidied this up so that it uses the producerMetrics created in setup plus just the single ProducerMetrics object created in this test case, rather than making two ProducerMetrics for the test case, and ignoring the one made in setup.